data-visualization/chart-types.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: page
+title:  Chart types
+category: Data visualization
+published: true
+---
+
+- [Bar or column charts](#bar-column-charts)
+- [Line charts](#line-charts)
+- [Pie or doughnut charts](#pie-doughnut-charts)
+ {: class="toc"}
+
+
+Choosing the correct chart for your data is important to avoid confusion and to
+accurately convey your data. For most graphs, stick with the basic chart types
+of bar and line graphs, since they are more commonly used and thus easier for
+people to understand.
+{: class="lead-in"}
+
+To help you choose what graph is best for your data, take a look at:
+[www.betterevaluation.org/sites/default/files/choosing-a-good-chart-09.pdf](www.betterevaluation.org/sites/default/files/choosing-a-good-chart-09.pdf)
+{: class="lead-in"}
+
+<h5>Key questions</h5>
+* Does the chart type accurately reflect the data? Would a different chart show
+the data better?
+* Is there an easily digestible amount of data in the chart? Does it need to be
+broken into more than one graphic?
+* Does the chart help users better understand the information?
+
+<h3 id="bar-column-charts">Bar or column charts</h3>
+Use bar or column charts to show comparisons of different discrete items,
+factors or categories. Comparisons could include items that can be counted and
+categorized. These are some of the easiest charts to interpret because the data
+is shown by length of bar.
+
+<h6>Key points</h6>
+* If exact numbers are important, the bars should be labeled with the value.
+* When possible label directly rather than relying on color.
+* For stacked charts, use a white line or stroke to create contrast between bar
+chunks.
+
+<h6>Examples</h6>
+
+<nomarkdown>
```

Comment:
Looking at the code, I think if you leave off the `nomarkdown` on images that aren't wrapped in a layout `div` and don't need to be wider than the paragraph text, you don't need the `br` tags because these images would be wrapped in a `p` and given the appropriate bottom margin.